sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,26 @@
+### 2019-08-06 5.0.0-preview.2
+
+#### General
+- The sender is refactored to avoid the `TooManyEventListeners` warning which would occur before if too many send requests were in flight at the same time from the same sender.
+- The receiver is refactored to allow the same underlying AMQP link to be shared between streaming and batching mode. This results in seamless transition between the three different receive methods on the `EventHubConsumer`
+- All time related entites have been updated to use milli seconds as the unit of time for consistency.
+- New error `InsufficientCreditError` is introduced for the scenario where [rhea](https://www.npmjs.com/package/rhea) is unable to send events due to its internal buffer being full. This is a transient error and so is treated as retryable.
+- The error `OperationTimeoutError` was previously mistakenly classified as an AMQP error which is now corrected. Since this can also be a transient error, it is treated as retryable.
+
+#### Publishing events
+- Introduced the `EventDataBatch`, allowing for publication of a batch of events with known size constraint. 
+  This allows publishers to build batches without the potential for the `MaxSizeExceededError` error when sending events. It also allows publishers with bandwidth concerns to control the size of each batch published.
+- Enhanced the `EventHubProducer` to allow creation of an `EventDataBatch` and to accept them for publication.
```

Comment:
We can remove this and as https://github.com/Azure/azure-sdk-for-js/pull/4675/files#r310844742 will cover the same intent

```suggestion
```